### ISSUE TITLE
Ability to zoom the preview window in the design section

### ISSUE THREAD

**Issue by kevmodrome** [Budibase/budibase#2345]
**Describe the feature request**
Sometimes you need to work on smaller screens, having the ability to change the "zoom" level of the preview window would be stellar. Just a simple dropdown somewhere would make sense to me.

**Comment by aptkingston** [comment]
This would tie in nicely with having the ability to choose the device to preview on too! Like mobile, tablet, desktops etc.

**Comment by aptkingston** [comment]
Yep totally. The only reason it's there is because (like you said) there is nowhere for global app settings haha. Both zoom and device selection could be centered above the preview (like in Joe's designs), and components could be moved into the right bar. The global app settings need to go somewhere else, which could either be a new page (if we have enough settings to create content) or in a top bar modal like we used to have settings.
